src/styles/HeartDoodleDialogue.module.scss

```diff
@@ -0,0 +1,33 @@
+.dialogueButton {
+    // position: absolute;
```

Comment:
In general, try not to submit PR's with comments in code unless there is a specific reason. For example a comment explaining why something is happening, or a `TODO` comment.